packages/legend-manual-tests/src/__tests__/roundtrip-grammar/ProfilingGraphBuilding.test.ts

```diff
@@ -187,7 +187,7 @@ const profileRoundtrip = async (
   });
   if (options.debug) {
     log.info(
-      LogEvent.create(V1_ENGINE_EVENT.GRAMMAR_TO_JSON),
+      LogEvent.create('engine.grammar.grammar-to-json'),
```

Comment:
I'm just cleaning up the log events and thought we don't need the `V1_engine` level logging for now, especially since test is the only place we're doing this. I think we can add this back when we actually using this in the main app logic